tests/server/cluster/cluster_test.go

```diff
@@ -1562,6 +1562,7 @@ func TestTransferLeaderBack(t *testing.T) {
 	svr := leaderServer.GetServer()
 	rc := cluster.NewRaftCluster(ctx, svr.ClusterID(), syncer.NewRegionSyncer(svr), svr.GetClient(), svr.GetHTTPClient())
 	rc.InitCluster(svr.GetAllocator(), svr.GetPersistOptions(), svr.GetStorage(), svr.GetBasicCluster(), svr.GetHBStreams(), svr.GetKeyspaceGroupManager())
+	rc.SchedulingController = cluster.NewSchedulingController(ctx, rc.GetBasicCluster(), rc.GetOpts(), rc.GetRuleManager())
```

Comment:
How about putting the initialization  of `SchedulingController` into `InitCluster`? This way we can avoid one less function call and avoid forgetting

Reply:
It doesn't init RuleManager.

Reply:
Also put  the initialization of `RuleManager ` into InitCluster?

Reply:
done